build.gradle.kts

```diff
@@ -14,7 +14,7 @@ group = "no.bekk"
 version = "0.0.1"
 application {
     mainClass.set("no.bekk.ApplicationKt")
-    mainClassName = "no.bekk.ApplicationKt"
+    mainClassName = "no.bekk.ApplicationKt" // Need this while shadow plugin < 7
```

Comment:
Vet ikke 🤷  men det er jo veldig lett å teste

Reply:
Var ment å være en kommentar, mainClassName er "deprecated" men er påkrevd mens vi bruker shadow plugin < 7.
Vi kan ikke bumpe til shadow plugin 7 fordi den krever en nyere gradle versjon. Kunne jo evt. migrert til den, men det funker fint som det er 🤷

Reply:
Beklager det gikk litt fort for meg